packages/material-tailwind-react/src/types/components/input.ts

```diff
@@ -24,8 +24,11 @@ export type labelProps = {
 export type containerProps = {
   [key: string]: any;
 };
-export type shrink = boolean;
+export type iconProps = {
+  [key: string]: any;
+};
 export type className = string;
+export type shrink = boolean;
```

Comment:
I have got a missing import error I have fixed it using the type as `boolean` is this correct? @sajadevo